scikits/odes/sundials/cvode.pyx

```diff
@@ -678,7 +747,14 @@ cdef class CVODE:
             'max_nonlin_iters':
                 default = 0,
             'nonlin_conv_coef':
-                default = 0.
+                default = 0,
+            'old_api':
+                Values: True (default), False
```

Comment:
I went with making `old_api=False` so as not to break existing code, at some point we can switch `old_api` to `True`, and then at some point we can just remove the old api. If making people switch isn't an issue, then we could just jump to making `old_api=True`.